forloop_modules/queries/node_context_requests_backend.py

```diff
@@ -1183,6 +1183,12 @@ def process_api_adjustments(adjustments_dict, project_uid):
 
     return response
 
+def get_user_log_messages(project_uid: str):
+    url = f'{BASE_API}/user_log_messages/{project_uid}'
+    response = requests.get(url)
```

Comment:
Please use `response.raise_for_status()` to throw exception in case the request was rejected. Otherwise it's much more troublesome to track the error.